InternetHospitalWebUI/src/app/Modules/routing/routing.module.ts

```diff
@@ -25,17 +15,20 @@ import { DoctorGuard } from '../../Services/Guards/doctor.guard';
 import { ModeratorGuard } from '../../Services/Guards/moderator.guard';
 import { AdminGuard } from '../../Services/Guards/admin.guard';
 
-import { ADMIN_PANEL } from '../../config';
+import { ADMIN_PANEL, DOCTOR_LIST } from '../../config';
 import { SIGN_IN } from '../../config';
 import { SIGN_UP } from '../../config';
 
 const ROUTES: Routes = [
   { path: '', component: HomeComponent },
   { path: SIGN_UP, component: SignUpComponent },
   { path: SIGN_IN, component: SignInComponent },
+  { path: '404', component: Page404Component },
```

Comment:
404 is also better to put in const, like PAGE_404/PAGE_NOT_FOUND. Because it looks weird, when all routs are consts and one is hardcode.